apps/nextjs/src/app/dashboard/games/[id]/[matchId]/_components/edit-team-dialog.tsx

```
@@ -0,0 +1,443 @@
+            roles: player.roles.map((role) => role.id),
+          };
+        }),
+      roles: teamRoles,
+    },
+  });
+  const onSubmit = (data: z.infer<typeof formSchema>) => {
+    const isSameTeamName = data.name === team.name;
+    const playersToAdd = data.players
+      .filter((player) => !originalTeamPlayers.find((p) => p.id === player.id))
+      .map((p) => {
+        const foundPlayer = players.find((p) => p.id === p.id);
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)